deploy/helm/templates/rbac.yaml

```diff
@@ -176,3 +176,40 @@ subjects:
     name: {{ include "trivy-operator.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }}
 {{- end }}
+
```

Comment:
we added couple of days ago support to ExposedSecretReport , we should add ClusterRole for it as well

Reply:
Done